include/openthread/platform/time.h

```diff
@@ -41,6 +41,11 @@
 extern "C" {
 #endif
 
+#define OT_MS_PER_S 1000    ///< Number of milliseconds per second
+#define OT_NS_PER_US 1000   ///< Number of nanoseconds per microsecond
+#define OT_US_PER_MS 1000   ///< Number of microseconds per millisecond
+#define OT_US_PER_S 1000000 ///< Number of microseconds per second
```

Comment:
Change order? So we start with milliseconds, then usec, andthen nan-seconds?
```suggestion
#define OT_MS_PER_S 1000    ///< Number of milliseconds per second
#define OT_US_PER_MS 1000   ///< Number of microseconds per millisecond
#define OT_US_PER_S 1000000 ///< Number of microseconds per second
#define OT_NS_PER_US 1000   ///< Number of nanoseconds per microsecond
```